### PR TITLE
Feature/portal component creation

### PR DESCRIPTION
**Pull request by benits**
This PR is for addition exportation of Portal component from chakra-ui.
Create testing and storybook;
upgrade of all libraries of chrakra-ui.

### CODE REVIEW

File: src/components/popover/PopoverArrow.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {
+  PopoverArrow as BasePopoverArrow,
+  PopoverArrowProps as BasePopoverArrowProps,
+} from '@chakra-ui/react';
+
+export { BasePopoverArrowProps as PopoverArrowProps };
+
+export const PopoverArrow: React.FC<BasePopoverArrowProps> = ({
```
Comment (KelvinCS) [inline src/components/popover/PopoverArrow.tsx:9]: Me lembra aqui. Usar o forwardRef aqui não rolou?

File: src/components/popover/Popover.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {
+  Popover as BasePopover,
+  PopoverProps as BasePopoverProps,
+} from '@chakra-ui/react';
+
+export { BasePopoverProps as PopoverProps };
+
+export const Popover: React.FC<BasePopoverProps> = ({ children, ...props }) => (
```
Comment (KelvinCS) [inline src/components/popover/Popover.tsx:9]: O forwardRef não rolou aqui?

File: src/components/popover/PopoverBody.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {
+  PopoverBody as BasePopoverBody,
+  PopoverBodyProps as BasePopoverBodyProps,
+} from '@chakra-ui/react';
+
+export { BasePopoverBodyProps as PopoverBodyProps };
+
+export const PopoverBody: React.FC<BasePopoverBodyProps> = ({
```
Comment (KelvinCS) [inline src/components/popover/PopoverBody.tsx:9]: forwardRef

File: src/components/popover/PopoverCloseButton.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {
+  PopoverCloseButton as BasePopoverCloseButton,
+  PopoverCloseButtonProps as BasePopoverCloseButtonProps,
+} from '@chakra-ui/react';
+
+export { BasePopoverCloseButtonProps as PopoverCloseButtonProps };
+
+export const PopoverCloseButton: React.FC<BasePopoverCloseButtonProps> = ({
```
Comment (KelvinCS) [inline src/components/popover/PopoverCloseButton.tsx:9]: forwardRef

File: src/components/popover/PopoverContent.tsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {
+  PopoverContent as BasePopoverContent,
+  PopoverContentProps as BasePopoverContentProps,
+} from '@chakra-ui/react';
+
+export { BasePopoverContentProps as PopoverContentProps };
+
+export const PopoverContent: React.FC<BasePopoverContentProps> = ({
```
Comment (KelvinCS) [inline src/components/popover/PopoverContent.tsx:9]: forwardRef

File: src/components/popover/PopoverFooter.tsx [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {
+  PopoverFooter as BasePopoverFooter,
+  PopoverFooterProps as BasePopoverFooterProps,
+} from '@chakra-ui/react';
+
+export { BasePopoverFooterProps as PopoverFooterProps };
+
+export const PopoverFooter: React.FC<BasePopoverFooterProps> = ({
```
Comment (KelvinCS) [inline src/components/popover/PopoverFooter.tsx:9]: forwardRef

File: src/components/popover/PopoverHeader.tsx [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {
+  PopoverHeader as BasePopoverHeader,
+  PopoverHeaderProps as BasePopoverHeaderProps,
+} from '@chakra-ui/react';
+
+export { BasePopoverHeaderProps as PopoverHeaderProps };
+
+export const PopoverHeader: React.FC<BasePopoverHeaderProps> = ({
```
Comment (KelvinCS) [inline src/components/popover/PopoverHeader.tsx:9]: forwardRef

File: src/components/popover/PopoverTrigger.tsx [hunk 8]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { PopoverTrigger as BasePopoverTrigger } from '@chakra-ui/react';
+
+export const PopoverTrigger: React.FC = ({ children, ...props }) => (
```
Comment (KelvinCS) [inline src/components/popover/PopoverTrigger.tsx:4]: forwardRef

File: src/components/portal/Portal.tsx [hunk 9]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {
+  Portal as BasePortal,
+  PortalProps as BasePortalProps,
+} from '@chakra-ui/react';
+
+export { BasePortalProps as PortalProps };
+
+export const Portal: React.FC<BasePortalProps> = ({ children, ...props }) => (
```
Comment (KelvinCS) [inline src/components/portal/Portal.tsx:9]: forwardRef

File: src/components/portal/PortalManager.tsx [hunk 10]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {
+  PortalManager as BasePortalManager,
+  PortalManagerProps as BasePortalManagerProps,
+} from '@chakra-ui/react';
+
+export { BasePortalManagerProps as PortalManagerProps };
+
+export const PortalManager: React.FC<BasePortalManagerProps> = ({
```
Comment (KelvinCS) [inline src/components/portal/PortalManager.tsx:9]: forwardRef

File: src/components/popover/PopoverArrow.tsx (hunk 1 above)
Comment (benits) [reply src/components/popover/PopoverArrow.tsx:9]: não, só foi possivel fowardRef em alguns components

**Comment by benits** [comment]
@KelvinCS adjustments in all suggestions, where it was possible to add fowardRef added it.
too it has added AlertDialog with testing and storybook.

this PR is ready for your review and approval.

**Review by KelvinCS — APPROVED** [review]

**Pull request merged**